scripts/test/index.js

```diff
@@ -0,0 +1,9 @@
+const { spawn } = require('child_process')
+const { inputDir = './source' } = require('yargs').argv
+
+const args = [
+  require.resolve('mocha/bin/_mocha'),
+  `${inputDir}/**/__tests__/*-test.js`
```

Comment:
It also sets up babel compilation with some default presets (I'd like to get them from the web pack config... but that might not be so straight-forward)